payments/service.go

```diff
@@ -0,0 +1,4 @@
+package payments
+
+// Service - struct definition of payments service
+type Service struct{}
```

Comment:
I'm not as familiar with go-lang. Is this effectively defining a `Service` type which is like an `any` type in Typescript?

Reply:
yes, currently this service has no attributes of its own, but when merged will have attributes from the other PRs associated.